sql-statements/sql-statement-explain-analyze.md

```diff
@@ -82,15 +82,177 @@ EXPLAIN ANALYZE SELECT * FROM t1;
+
+### IndexHashJoin
+
+The execution process of the ʻIndexHashJoin` operator is similar to the ʻIndexJoin` operator. `IndexHashJoin` operator also has 1 outer worker and N inner workers to executed in parallel, but the output order is not promised. The specific execution process is as follows:
+
+1. The outer worker reads N outer rows, builds a task and sends it to the
+inner worker channel.
+2. The inner worker receives the tasks and does 3 things for every task:
+   a. builds hash table from the outer rows
+   b. builds key ranges from outer rows and fetches inner rows
+   c. probes the hash table and sends the join result to the result channel.
+   Note: step a and step b are running concurrently.
+3. The main thread of ʻIndexHashJoin`  receives the join results from the result channel.
+
+The ʻIndexHashJoin` operator contains the following execution information:
+
+```sql
+inner:{total:4.429220003s, concurrency:5, task:17, construct:96.207725ms, fetch:4.239324006s, build:24.567801ms, join:93.607362ms}
+```
+
+-ʻInner`: the execution information of inner worker:
+    - total: the total time-consuming by the inner worker.
+    - concurrency: the number of inner workers.
+    - task: The total number of tasks processed by the inner worker.
+    - construct: the prepare time before the inner worker reads the inner table rows.
+    - fetch: The total time-consuming of inner worker to read inner table rows.
+    - Build: The total time-consuming of inner worker to construct the hash map of the outer table rows.
+    - join:  The total time-consuming of inner worker to do join with the inner table rows and the hash map of outer table rows.
+
+### HashJoin
+
+The `HashJoin` operator has an inner worker, an outer worker and N join workers. The specific execution process is as follows:
+
+1. The inner worker reads inner table rows and constructs a hash map.
+2. The outer worker reads the outer table rows, then wraps it into a task and sends it to the join worker.
+3. Wait for the completion of the hash map construction in step 1.
+4. The join worker uses the outer table rows and hash map in the task to do join, and then sends the join result to the result channel.
+5. The main thread of `HashJoin` receives the join result from the result channel.
+
+The `HashJoin` operator contains the following execution information:
+
+```
+build_hash_table:{total:146.071334ms, fetch:110.338509ms, build:35.732825ms}, probe:{concurrency:5, total:857.162518ms, max:171.48271ms, probe:125.341665ms, fetch:731.820853ms}
+```
+
+- `build_hash_table`: Read the data of the inner table and construct the execution information of the hash map:
+    - total: total time spent.
+    - fetch: The total time spent reading inner table data.
+    - build: The total time spent constructing a hash map.
+-probe: execution information of join worker:
```

Comment:
This is not rendering correctly. I think it just needs a small change, but I can't easily preview just this:
```suggestion
- probe: execution information of join worker:
```